astropy/visualization/mpl_style.py

```diff
@@ -1,23 +1,21 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-
-__doctest_requires__ = {'*': ['matplotlib']}
-
 """ This module contains dictionaries that can be used to set a
 matplotlib plotting style.
+
 It is mostly here to allow a consistent plotting style in tutorials,
 but can be used to prepare any matplotlib figure.
 
-Using a matplotlib version > 1.4 you can do::
+Using a matplotlib version >= 1.5 you can do::
 
-    >>> import matplotlib.pyplot as plt
-    >>> from astropy.visualization import astropy_mpl_style
-    >>> plt.style.use(astropy_mpl_style)
```

Comment:
If I remember correctly there should be an indented block after `::`. 

I wouldn't rely on it working without indentation (see for example http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_syntax.html#inserting-code-and-literal-blocks)